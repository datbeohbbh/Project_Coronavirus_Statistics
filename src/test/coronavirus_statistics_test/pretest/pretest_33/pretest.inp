50 100
q 1 48 189
q 1 29 165
? 4 1
q 1 8 265
? 6
? 4 3
q 2 29 17
q 3 8 248
? 4 2
q 1 36 10
q 1 45 76
q 1 35 441
q 3 35 307
? 4 3
q 1 17 413
q 1 11 162
q 2 17 360
q 1 4 476
q 1 41 262
? 7
? 7
q 1 27 243
? 4 6
q 1 14 316
? 4 6
? 4 1
q 1 16 12
? 6
q 1 42 129
q 2 16 11
q 1 20 448
q 2 27 183
? 7
q 3 16 1
q 1 7 480
? 4 2
q 1 16 316
q 2 14 293
q 1 9 358
q 3 29 24
? 4 1
q 1 44 194
? 6
? 6
? 6
q 2 11 93
? 6
q 2 4 2
? 6
? 4 2
q 2 17 46
? 6
q 2 16 270
? 4 8
q 2 48 42
q 2 7 467
? 7
q 1 10 464
? 4 5
? 4 3
? 4 6
? 6
q 2 41 58
q 2 17 7
? 4 12
? 4 5
? 4 6
q 1 28 288
q 2 36 10
? 6
? 4 18
q 1 17 26
? 6
? 4 6
q 1 39 261
q 3 16 7
? 6
q 1 36 454
q 3 17 22
q 2 35 114
? 4 8
q 2 28 113
q 3 16 36
q 1 24 22
? 4 14
? 4 13
? 6
? 4 12
? 4 20
? 4 8
? 4 1
? 4 21
? 4 16
? 4 16
? 6
? 6
? 6
? 4 21
? 7
? 4 10