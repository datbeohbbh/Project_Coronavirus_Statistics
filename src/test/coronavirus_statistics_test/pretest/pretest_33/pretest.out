Query ? 4 1 :
The answer is the city with the index 29 
- Information of the city 29 -
-> Number of current patients : 165
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
Currently, no recovery cases have been recorded

Query ? 4 3 :
The answer is the city with the index 8 
- Information of the city 8 -
-> Number of current patients : 265
-> Has recovered : 0
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 29 
- Information of the city 29 -
-> Number of current patients : 148
-> Has recovered : 17
-> Was dead : 0

Query ? 4 3 :
The answer is the city with the index 45 
- Information of the city 45 -
-> Number of current patients : 76
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 35
Quantity: 307 people

Query ? 7 :
The city with the highest death is city with the index 35
Quantity: 307 people

Query ? 4 6 :
The answer is the city with the index 29 
- Information of the city 29 -
-> Number of current patients : 148
-> Has recovered : 17
-> Was dead : 0

Query ? 4 6 :
The answer is the city with the index 29 
- Information of the city 29 -
-> Number of current patients : 148
-> Has recovered : 17
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 36 
- Information of the city 36 -
-> Number of current patients : 10
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 17
Quantity: 360 people

Query ? 7 :
The city with the highest death is city with the index 35
Quantity: 307 people

Query ? 4 2 :
The answer is the city with the index 8 
- Information of the city 8 -
-> Number of current patients : 17
-> Has recovered : 0
-> Was dead : 248

Query ? 4 1 :
The answer is the city with the index 36 
- Information of the city 36 -
-> Number of current patients : 10
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 17
Quantity: 360 people

Query ? 6 :
The city with the highest recovery is city with the index 17
Quantity: 360 people

Query ? 6 :
The city with the highest recovery is city with the index 17
Quantity: 360 people

Query ? 6 :
The city with the highest recovery is city with the index 17
Quantity: 360 people

Query ? 6 :
The city with the highest recovery is city with the index 17
Quantity: 360 people

Query ? 4 2 :
The answer is the city with the index 8 
- Information of the city 8 -
-> Number of current patients : 17
-> Has recovered : 0
-> Was dead : 248

Query ? 6 :
The city with the highest recovery is city with the index 17
Quantity: 406 people

Query ? 4 8 :
The answer is the city with the index 45 
- Information of the city 45 -
-> Number of current patients : 76
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 35
Quantity: 307 people

Query ? 4 5 :
The answer is the city with the index 14 
- Information of the city 14 -
-> Number of current patients : 23
-> Has recovered : 293
-> Was dead : 0

Query ? 4 3 :
The answer is the city with the index 7 
- Information of the city 7 -
-> Number of current patients : 13
-> Has recovered : 467
-> Was dead : 0

Query ? 4 6 :
The answer is the city with the index 16 
- Information of the city 16 -
-> Number of current patients : 46
-> Has recovered : 281
-> Was dead : 1

Query ? 6 :
The city with the highest recovery is city with the index 7
Quantity: 467 people

Query ? 4 12 :
The answer is the city with the index 48 
- Information of the city 48 -
-> Number of current patients : 147
-> Has recovered : 42
-> Was dead : 0

Query ? 4 5 :
The answer is the city with the index 16 
- Information of the city 16 -
-> Number of current patients : 46
-> Has recovered : 281
-> Was dead : 1

Query ? 4 6 :
The answer is the city with the index 27 
- Information of the city 27 -
-> Number of current patients : 60
-> Has recovered : 183
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 7
Quantity: 467 people

Query ? 4 18 :
The answer is the city with the index 4 
- Information of the city 4 -
-> Number of current patients : 474
-> Has recovered : 2
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 7
Quantity: 467 people

Query ? 4 6 :
The answer is the city with the index 27 
- Information of the city 27 -
-> Number of current patients : 60
-> Has recovered : 183
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 7
Quantity: 467 people

Query ? 4 8 :
The answer is the city with the index 11 
- Information of the city 11 -
-> Number of current patients : 69
-> Has recovered : 93
-> Was dead : 0

Query ? 4 14 :
The answer is the city with the index 28 
- Information of the city 28 -
-> Number of current patients : 175
-> Has recovered : 113
-> Was dead : 0

Query ? 4 13 :
The answer is the city with the index 48 
- Information of the city 48 -
-> Number of current patients : 147
-> Has recovered : 42
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 7
Quantity: 467 people

Query ? 4 12 :
The answer is the city with the index 42 
- Information of the city 42 -
-> Number of current patients : 129
-> Has recovered : 0
-> Was dead : 0

Query ? 4 20 :
The answer is the city with the index 36 
- Information of the city 36 -
-> Number of current patients : 454
-> Has recovered : 10
-> Was dead : 0

Query ? 4 8 :
The answer is the city with the index 27 
- Information of the city 27 -
-> Number of current patients : 60
-> Has recovered : 183
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 16 
- Information of the city 16 -
-> Number of current patients : 3
-> Has recovered : 281
-> Was dead : 44

Query ? 4 21 :
The answer is the city with the index 10 
- Information of the city 10 -
-> Number of current patients : 464
-> Has recovered : 0
-> Was dead : 0

Query ? 4 16 :
The answer is the city with the index 41 
- Information of the city 41 -
-> Number of current patients : 204
-> Has recovered : 58
-> Was dead : 0

Query ? 4 16 :
The answer is the city with the index 41 
- Information of the city 41 -
-> Number of current patients : 204
-> Has recovered : 58
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 7
Quantity: 467 people

Query ? 6 :
The city with the highest recovery is city with the index 7
Quantity: 467 people

Query ? 6 :
The city with the highest recovery is city with the index 7
Quantity: 467 people

Query ? 4 21 :
The answer is the city with the index 10 
- Information of the city 10 -
-> Number of current patients : 464
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 35
Quantity: 307 people

Query ? 4 10 :
The answer is the city with the index 45 
- Information of the city 45 -
-> Number of current patients : 76
-> Has recovered : 0
-> Was dead : 0

